packages/server/manage_container/client/manage_to_db/client.go

```diff
@@ -326,3 +328,19 @@ func (c Client) GetMatchingColumn(dataID string) (int32, error) {
 
 	return data.Meta.MatchingColumn, nil
 }
+
+func (c Client) CreateStatusFile(jobUUID string) {
+	ls.Lock(jobUUID)
+	defer ls.Unlock(jobUUID)
+	path := fmt.Sprintf("%s/%s", resultDbPath, jobUUID)
+	os.Mkdir(path, 0777)
+	fp, _ := os.Create(fmt.Sprintf("%s/status_%s", path, pb_types.JobStatus(2).String()))
+	fp.Close()
+}
+
+func (c Client) DeleteStatusFile(jobUUID string) {
+	ls.Lock(jobUUID)
+	defer ls.Unlock(jobUUID)
+	path := fmt.Sprintf("%s/%s", resultDbPath, jobUUID)
+	os.RemoveAll(path)
```

Comment:
Use that return value as it may return an error.

- https://pkg.go.dev/os#RemoveAll

```suggestion
	return os.RemoveAll(path)
```

Reply:
@arukuka 
Same as above